upf/upf_pfcp.c

```diff
@@ -1020,6 +1021,17 @@ pfcp_disable_session (upf_session_t * sx)
   return 0;
 }
 
+void
+upf_delete_nat_binding (upf_session_t * sx)
+{
+  upf_nat_addr_t *ap = sx->nat_addr;
+  if (ap->used_blocks > 0)
```

Comment:
calling this with `ap->used_blocks == 0` would be critical error, right?
Then maybe this should be an assert,

Reply:
added